src/lib/components/sections/navbar.svelte

```diff
@@ -205,6 +205,9 @@
 
           a {
             font-size: var(--size-md);
+            display: block;
+            padding: 30px 900px;
+            margin-top: -50px;
```

Comment:
how come margin top here? make it asymmetrical i think
<img width="1371" alt="image" src="https://user-images.githubusercontent.com/8981287/190535715-687103bb-b04a-4c26-af3e-c3625c369cbd.png">

Reply:
I shall try and fix this!